lib/humidifier/aws_adapters/base.rb

```diff
@@ -31,6 +31,12 @@ def update(payload)
         try_valid { client.update_stack(payload.update_params) }
       end
 
+      # Upload a CFN stack to S3 so that it can be referenced via template_url
+      def upload(payload)
+        Humidifier.config.ensure_upload_configured!(payload)
+        upload_object(payload, "#{Humidifier.config.s3_prefix}#{payload.identifier}.json")
```

Comment:
s3_prefix is completely configured by the user. I could see a case where you might have two apps using Humidifier and want something like `humidifier/app1-` as the prefix, so I don't think we can guess what they mean necessarily.